src/stores/notification_store.ts

```diff
@@ -1,9 +1,28 @@
-import { createAbstractStore } from "../store_engine";
 import { InformationNotification } from "../types";
 
-export interface NotificationStore {
-  notifyUser: (notification: InformationNotification) => any;
-  raiseError: (text: string, callback?: () => void) => any;
-  askConfirmation: (content: string, confirm: () => any, cancel?: () => any) => any;
+interface NotificationStoreMethods {
+  notifyUser: (notification: InformationNotification) => void;
+  raiseError: (text: string, callback?: () => void) => void;
+  askConfirmation: (content: string, confirm: () => void, cancel?: () => void) => void;
+}
+
+export class NotificationStore {
+  notifyUser: NotificationStoreMethods["notifyUser"] = (notification) => alert(notification);
```

Comment:
```suggestion
  notifyUser: NotificationStoreMethods["notifyUser"] = (notification) => window.alert(notification);
```

for consistency (and ease of reading)